pkg/configuration/parser.go

```diff
@@ -329,10 +304,13 @@ func (p *nsxConfigParser) getEndpointsFromGroupsPaths(groupsPaths []string, excl
 		}
 	} else {
 		for _, ip := range ips {
-			ruleBlock := p.topology.allRuleIPBlocks[ip]
-			res.VMs = append(res.VMs, ruleBlock.VMs...)
-			res.VMs = append(res.VMs, ruleBlock.ExternalIPs...)
-			res.Blocks = append(res.Blocks, ruleBlock)
+			if ruleBlock := p.topology.allRuleIPBlocks[ip]; ruleBlock != nil {
+				res.VMs = append(res.VMs, ruleBlock.VMs...)
+				res.VMs = append(res.VMs, ruleBlock.ExternalIPs...)
+				res.Blocks = append(res.Blocks, ruleBlock)
+			} else {
+				logging.InternalErrorf("didint find block of ip %s", ip)
```

Comment:
can you explain this error?

Reply:
removing it.
its happaned when a path which is neither group nore valid IP.
however, I warn such case when reading topology: